### ISSUE TITLE
about performance

### ISSUE THREAD

**Issue by pengfeidip** [xiongzihua/pytorch-YOLO-v1#14]
我用原始代码训练, vgg16作为backbone, 50个epoch(原始参数)后  mAP达到**44.6%**, 训练70个epoch后达到 **49.8%** ... 从数据上看, 没啥问题, 但是不知道为什么距离你的readme里面还有很大的距离. 

![image](https://user-images.githubusercontent.com/23066797/50408604-0c065880-0827-11e9-9931-6216c83cd941.png)
（50 epoches, left: training loss right: val loss）

![image](https://user-images.githubusercontent.com/23066797/50408619-22acaf80-0827-11e9-8f05-e5e59ddfff24.png)
  (70 epoches, left: training loss right: val loss)

----
I trained the network with the original code,  vgg16 as the backbone,. after 50 epoches (original parameters) mAP is 44.6%. After  70 epoches , mAP is 49.8% ... Why does this not achieve the performance inside your readme.

**Comment by pengfeidip** [comment]
暂时不需要回复, 我好像发现了问题, 等我整理再总结一下...

**Comment by myhooo** [comment]
@pengfeidip 您好，时间隔的有点久， 我也遇到这种情况,mAP很小，请问您后来发现是什么问题呀？

**Comment by pengfeidip** [comment]
> @pengfeidip 您好，时间隔的有点久， 我也遇到这种情况,mAP很小，请问您后来发现是什么问题呀？

@myhooo 抱歉，太久了，我已经忘了....

**Comment by myhooo** [comment]
> > @pengfeidip 您好，时间隔的有点久， 我也遇到这种情况,mAP很小，请问您后来发现是什么问题呀？
> 
> @myhooo 抱歉，太久了，我已经忘了....

仍然感谢您的回复！如果您仍从事相关的研究的话，请问一般训练的mAP比较低，我训练50epoch的时候，resnet才能达到43%，可以考虑是哪些方面的问题呀？
